src/java.base/windows/native/libjava/jni_util_md.c

```diff
@@ -36,31 +36,6 @@ void* getProcessHandle() {
     return (void*)GetModuleHandle(NULL);
 }
 
-/*
- * Windows doesn't have an RTLD_DEFAULT equivalent, so in stead we have to
- * iterate over all the modules loaded by the process to implement the
- * default library behaviour.
- */
-void* findEntryInProcess(const char* name) {
```

Comment:
I think removing this also allows the removal of `#include <psapi.h>` from the includes. I'll give it a try here as well.

Reply:
Confirmed, removing the include is safe.

Reply:
Thanks. Will remove that include.